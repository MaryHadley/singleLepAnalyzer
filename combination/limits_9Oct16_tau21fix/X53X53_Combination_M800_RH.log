000 (1295374.000.000) 10/09 22:41:05 Job submitted from host: <192.168.63.249:41533>
...
001 (1295374.000.000) 10/09 22:41:26 Job executing on host: <192.168.63.241:42920>
...
006 (1295374.000.000) 10/09 22:41:34 Image size of job updated: 188480
	86  -  MemoryUsage of job (MB)
	87756  -  ResidentSetSize of job (KB)
...
006 (1295374.000.000) 10/09 22:46:34 Image size of job updated: 282496
	144  -  MemoryUsage of job (MB)
	146644  -  ResidentSetSize of job (KB)
...
006 (1295374.000.000) 10/10 00:01:43 Image size of job updated: 282496
	144  -  MemoryUsage of job (MB)
	146656  -  ResidentSetSize of job (KB)
...
006 (1295374.000.000) 10/10 20:53:53 Image size of job updated: 297820
	158  -  MemoryUsage of job (MB)
	160940  -  ResidentSetSize of job (KB)
...
005 (1295374.000.000) 10/10 21:01:48 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 22:17:39, Sys 0 00:00:40  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 22:17:39, Sys 0 00:00:40  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1590  -  Run Bytes Sent By Job
	306  -  Run Bytes Received By Job
	1590  -  Total Bytes Sent By Job
	306  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31769        1   3824814
	   Memory (MB)          :      158     3072      3072
...
