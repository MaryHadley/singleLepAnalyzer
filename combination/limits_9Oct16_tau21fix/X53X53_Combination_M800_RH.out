/var/lib/condor/execute/dir_5788
Total number of likelihood evaluations: 1100000100
Total number of likelihood evaluations: 11000100
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M800_RH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M800_RH/quant-toys:0-sig-4358d80c83.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M800_RH/quant-data-sig-bf752156b6.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M800_RH
