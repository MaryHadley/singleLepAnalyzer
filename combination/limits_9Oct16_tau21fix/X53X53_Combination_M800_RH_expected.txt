# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0    0.08651   0.043132    0.18245  0.059439    0.12807
